core/src/test/java/org/bitcoinj/utils/UtilsTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2017 Nicola Atzei
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bitcoinj.utils;
```

Comment:
I just noticed we already have an `org/bitcoinj/core/UtilsTest.java`, so there is no need to create an additional class in a different package. You can just add your methods to that test class.